about.html

```diff
@@ -0,0 +1,232 @@
+        </figure>
+      </div>
+    </section>
+    <hr />
+    <section class="history">
+      <div class="cont-history">
+        <h1 class="main-title history-title">See the past WE Global Summits</h1>
+        <p class="text-history">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellat
+          numquam sit ducimus!
+        </p>
+
+        <div class="cont-img-history">
+          <figure class="history-img-cont">
+            <img
+              src="./images/river.jpg"
+              alt="logo"
+              class="summit-history-img"
+            />
+
+            <div class="cont-hist-text">
+              <div class="hist-text-wrapper">
+                <h1 class="heading-date">2019</h1>
+                <p class="img-text">WE Global Summits in Africa</p>
+              </div>
+            </div>
+          </figure>
+          <span class="separator">separator</span>
+
+          <figure class="history-img-cont">
+            <img
+              src="./images/river.jpg"
+              alt="logo"
+              class="summit-history-img"
+            />
+            <div class="cont-hist-text">
+              <div class="hist-text-wrapper">
+                <h1 class="heading-date">2020</h1>
+                <p class="img-text">WE Global Summits in Europe</p>
+              </div>
+            </div>
+          </figure>
+        </div>
+      </div>
+    </section>
+
+    <section class="partner">
+      <h1 class="partner-heading">Partner</h1>
+      <ul class="partners-cont">
+        <li class="single-partner-cont moz">mozilla</li>
+        <li class="single-partner-cont">
+          <i class="fab fa-google fa-3x logo-ggl"></i>
+          <span class="ggl">oogle</span>
+        </li>
+        <li class="single-partner-cont nav">NAVER</li>
+        <li class="single-partner-cont daum">daumkakao</li>
+        <li class="single-partner-cont">
+          <i class="fab fa-airbnb logo-bnb fa-3x"></i>
+          <span class="airbnb">airbnb</span>
+        </li>
+      </ul>
+    </section>
+
+    <footer class="footer">
+      <div class="cont-elm-footer">
+        <figure class="logo-img-cont">
+          <img src="./images/logo.png" alt="logo" class="logo-footer" />
+          <div class="logo-text">
+            <h2 class="website-name">Earth</h2>
+            <h2 class="website-name">Planet</h2>
+          </div>
+        </figure>
+        <div class="footer-text-cont">
+          <h5 class="copyright-part">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          </h5>
+          <p class="society-social-part">
+            Lorem ipsum dolor sit amet consectetur adipisicing.
+            <a href="#" class="url-cont"
+              >https://www.behance.net/gallery/29845175/CC-Global-Summit-2021</a
+            >
+            Lorem ipsum dolor sit amet.
+          </p>
+          <p class="copyright-owner">&copy; Copyright Earth Planet 2021</p>
+        </div>
+      </div>
```

Comment:
- [ ] This part in the `about` page should not be visible in the desktop version
![Screenshot from 2021-10-15 19-37-03](https://user-images.githubusercontent.com/62705947/137529851-0e30cfc2-33b3-42d4-bae5-a6e00131ff13.png)